ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusSliderContentView.swift

```diff
@@ -144,28 +145,33 @@ struct SwipeDifferenceView<Leading: View, Trailing: View>: View {
         }
     }
     .onAppear {
-      prepareSliderHaptics()
-
-      Task.delayed(bySeconds: 2.5) { @MainActor in
-        hapticsTimer = Timer.scheduledTimer(withTimeInterval: 0.25, repeats: true) { _ in
-          createRandomHapticEffect()
+      Task.delayed(bySeconds: 1.5) { @MainActor in
+        withAnimation(.easeInOut(duration: 1.0)) {
+          prepareSliderHaptics()
+          prepareRandomHaptics()
         }
-        hapticsTimer?.tolerance = 0.1
       }
     }
     .onDisappear(perform: {
-      hapticsTimer?.invalidate()
-
-      Task { @MainActor in
-        await stopAllHaptics()
-      }
+      stopAllHaptics()
     })
     .onChange(of: progress) { newValue in
       hapticsLevel = determineHapticIntensityLevel(from: newValue)
     }
     .onChange(of: hapticsLevel) { newValue in
       createContinousHapticFeedback(intensity: newValue)
     }
+    .onChange(of: scenePhase) { newPhase in
+      switch scenePhase {
```

Comment:
You should use `newValue` here

Reply:
Changed with https://github.com/brave/brave-core/pull/23471/commits/278fd54efc2b46571485d77692db3b63d26df0ec